storage/src/main/java/org/commonjava/storage/pathmapped/core/PathMappedFileManager.java

```diff
@@ -71,9 +72,16 @@ public InputStream openInputStream( String fileSystem, String path ) throws IOEx
         String storageFile = pathDB.getStorageFile( fileSystem, path );
         if ( storageFile == null )
         {
-            throw new IOException( "Could not open input stream to for path {}-{}: path-mapped physical file does not exist." );
+            throw new IOException(
+                    "Could not open input stream to for path {}-{}: path-mapped physical file does not exist." );
+        }
+        final InputStream stream = physicalStore.getInputStream( storageFile );
+        if ( stream == null )
+        {
+            throw new IOException(
+                    "Could not open input stream to for path {}-{}: path-mapped physical file does not exist or the path is a directory." );
```

Comment:
Sems like the params in the string ("{}") are not replaced by any values and the same is in the exception above..

Reply:
Damn.... copy-paste problem again. That's why I was told not to become used to copy-paste... :cry: